src/main/kotlin/com/autonomousapps/internal/utils/gradleStrings.kt

```diff
@@ -69,7 +69,7 @@ internal fun ResolvedArtifactResult.toCoordinates(): Coordinates {
 
   return IncludedBuildCoordinates.of(
     requested = requested,
-    resolvedProject = identity
+    resolvedProject = resolved
```

Comment:
Preserve the "real" project coordinate here. `identity` is an artificial project coordinate that does not really exists in Gradle. See also adjustment and comment on `IncludedBuildProject.groovy` test fixture.